tests/widget/inline/widget-inline.stage-attribute.test.js

```diff
@@ -1,5 +1,4 @@
 import { expect, test } from '@playwright/test';
-
```

Comment:
For future reference, when you see changed files like this, and the change is just whitespace, please revert these files to the original state. It just adds a bunch of noise to the PR that can distract from your work.